blocks/banner/banner.js

```diff
@@ -0,0 +1,76 @@
+import {
+  div, button, h2, h3, form, input, a,
+} from '../../scripts/dom-helpers.js';
+
+function createLinkElement(hrefValue, innerHTML) {
+  const link = a({ class: 'spotlight-link', href: hrefValue }, innerHTML);
+
+  return link;
+}
+
+function createButton(hrefValue) {
+  const spotlightButton = button({
+    class: 'spotlight-button',
+    onclick: () => { window.location.href = hrefValue; },
+  }, 'LEARN MORE');
+
+  return spotlightButton;
+}
+
+function createBannerText(block) {
+  // extract text values from block elements
+  const spotlightTitle = block.querySelector('h2');
+  const spotlightSubtitle = block.querySelector('h3');
+
+  // create hyperlinks with the text values
+  const spotlightLink = block.querySelector('a');
+  const hrefValue = spotlightLink ? spotlightLink.href : '#';
+  const titleLink = createLinkElement(hrefValue, spotlightTitle.innerHTML);
+  const subtitleLink = createLinkElement(hrefValue, spotlightSubtitle.innerHTML);
+
+  spotlightTitle.remove();
+  spotlightSubtitle.remove();
+  spotlightLink.remove();
+
+  // style the hyperlinks
+  const heading = h2({ class: 'spotlight-text title' }, titleLink);
+  const subheading = h3({ class: 'spotlight-text subtitle' }, subtitleLink);
+
+  // create the button
+  const spotlightButton = createButton(hrefValue);
+
+  // create the form element
+  const spotlightInput = input({ class: 'spotlight-input', placeholder: 'Enter Email' });
+  const spotlightFormButton = button({ class: 'spotlight-form-button' }, 'SUBMIT');
+  const spotlightForm = form({ class: 'spotlight-form' }, spotlightInput, spotlightFormButton);
+
+  const textDiv = div({
+    class: 'spotlight-text',
```

Comment:
This class on the div doesn't look right.  The h2 and h3 probably should have it.

Reply:
I have renamed the `spotlight-text` to `spotlight-text-container` for better clarity.